src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -32,6 +32,29 @@ namespace Dynamo.LibraryViewExtensionWebView2
 {
     [PermissionSet(SecurityAction.Demand, Name = "FullTrust")]
     [ComVisibleAttribute(true)]
+
+    public class ScriptObject
```

Comment:
comments for public class and methods please, also please confirm if they need to be public or could be internal

Reply:
They have to be public as we access them through `webview.hostObjects.scriptObject` otherwise they wouldn't be usable from the Library component.